benefits/enrollment/templates/enrollment/success.html

```diff
@@ -7,26 +7,45 @@
   {{ block.super |add:" enrollment-success" }}
 {% endblock classes %}
 
+{% block page_title %}
+  {% translate "enrollment.pages.success.title" %}&nbsp;|&nbsp;
+{% endblock page_title %}
+
 {% block headline %}
-  <div class="col-lg-10">
-    <h1 class="pb-lg-8 pb-4">{{ page.headline }}</h1>
+  <div class="col-lg-8">
+    <h1 class="pb-lg-5 pb-4">{% translate "enrollment.pages.success.headline" %}</h1>
   </div>
 {% endblock headline %}
 
 {% block inner-content %}
-  <div class="col-12 col-sm-12 col-lg-10">{% include "core/includes/media-list.html" with media=media %}</div>
+  <div class="col-12 col-sm-12 col-lg-9">
+    <div class="row flex-column-reverse flex-lg-row">
+      <div class="col-12 col-lg-6">
+        <p class="pt-lg-4 mt-lg-3">{% translate "enrollment.pages.success.login_gov.confirm_item.details" %}</p>
+        <p class="pt-4">{% translate "enrollment.pages.success.thankyou" %}</p>
+      </div>
+      <div class="col-12 offset-lg-1 col-lg-5">
+        <img width="180"
+             height="155.1"
+             class="illo d-block mx-auto mb-4 mb-lg-0"
+             src="{% get_static_prefix %}img/enrollment-success-illustration.png"
+             alt="" />
```

Comment:
Note: I confirmed with @srhhnry that this image has _blank_ alt text. The text is not necessary for the user and in fact, it would slow the user's experience down in completing the whole flow.

Reply:
> it would slow the user's experience down in completing the whole flow

Isn't this the final image when the user is done with their flow?

Disagree, we need alt text here.

Reply:
<img width="346" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/95b24d7c-c8fe-48e1-9e45-72ac0af7a438">
This is what @srhhnry says.

Should we wait on someone to write alt text for this in the meantime, or can I just put a `TODO` placeholder in the meantime?

Reply:
Hmm, I disagree -- according to this reasoning, the homepage image doesn't need alt text either? Actually, none of our images (icons) would, as they aren't strictly required for moving the user through the flow, they are all decorative and in addition to the text we display.

I'll create an issue so we don't block this PR.

Reply:
We do have this from MobiMart: `alt="A woman and child paying for bus fare from their smart phone while at bus stop"` but maybe the alt tag could be changed to specify that the riders are using their benefit(???)

Reply:
#1514 assigned to @indexing for when he is back.